src/visitors/fs.js

```diff
@@ -31,9 +37,37 @@ module.exports = {
         __dirname: Path.dirname(asset.name),
         __filename: asset.basename
       };
-      let [filename, ...args] = path
-        .get('arguments')
-        .map(arg => evaluate(arg, vars));
```

Comment:
Maybe instead of changing the logic to be a loop that exists early as below, you could just wrap this in a `try...catch` block and do the logging in the catch. That way you'd also not need the `nodeNotEvaluated` thing. Might clean up the code a bit.